### DISCUSSION TITLE
Where is this symbol described in Starship documentation?

### DISCUSSION THREAD

**Discussion by nzwart** [starship/starship#4782] [Q&A] [answered]
New to Starship. This star # symbol appears in my terminal after I ctrl+z, seemingly counting the number of times that a SIGTSTP has happened during this terminal session: <img width="71" alt="Screenshot 2022-12-31 at 8 29 58 AM" src="https://user-images.githubusercontent.com/98571651/210149775-2039ae3c-e02f-4217-b27d-03f5c7217b69.png">

But I can't find any info about the symbol in Starship's documentation. Could someone please point me in the right direction?

**Comment by davidkna** [comment, marked as answer]
This looks like the [jobs module](https://starship.rs/config/#jobs). To confirm, try running `starship explain --jobs 2`.

**Reply by nzwart** [reply]
That's it. Thank you for the clarification.
